repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/libraries/preparetransaction.py

```diff
@@ -163,3 +163,34 @@ def mount_dnf_cache(overlayfs_info):
 
 def umount_dnf_cache(overlayfs_info):
     return check_cmd_call(['/bin/umount', os.path.join(overlayfs_info.merged, 'var', 'cache', 'dnf')])
+
+
+def create_disk_image(path):
+    diskimage_path = os.path.join(path, 'diskimage')
+    mounts_path = os.path.join(path, 'mounts')
+
+    if os.path.isdir(mounts_path):
+        check_cmd_call(['/bin/umount', '-fl', mounts_path])
```

Comment:
You could give "remove_disk_image" function an "ignore_errors" parameter and then call it here.